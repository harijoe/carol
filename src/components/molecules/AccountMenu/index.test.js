import React from 'react'
import { shallow } from 'enzyme'

import MenuAccount from '.'
import intlMock from '../../../../test/intlMock'

const wrap = (props = {}) => shallow(intlMock(<MenuAccount logout={() => {}} {...props} />))

test('renders MenuAccount', () => {
  expect(wrap()).toMatchSnapshot()
})
