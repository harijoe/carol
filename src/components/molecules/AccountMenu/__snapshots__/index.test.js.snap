// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MenuAccount 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <AccountMenu
    logout={[Function]}
/>,
  "nodes": Array [
    <AccountMenu
      logout={[Function]}
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <IntlProvider
        locale="en"
        messages={Object {}}
>
        <AccountMenu
                logout={[Function]}
        />
</IntlProvider>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": IntlProvider {
        "_reactInternalInstance": [Circular],
        "context": Object {
          "intl": undefined,
        },
        "props": Object {
          "children": <AccountMenu
            logout={[Function]}
/>,
          "locale": "en",
          "messages": Object {},
        },
        "refs": Object {},
        "state": Object {
          "getDateTimeFormat": [Function],
          "getMessageFormat": [Function],
          "getNumberFormat": [Function],
          "getPluralFormat": [Function],
          "getRelativeFormat": [Function],
          "now": [Function],
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <AccountMenu
          logout={[Function]}
/>,
        "_debugID": 2,
        "_renderedOutput": <AccountMenu
          logout={[Function]}
/>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <IntlProvider
    locale="en"
    messages={Object {}}
>
    <AccountMenu
        logout={[Function]}
    />
</IntlProvider>,
}
`;
